10.16.0.17 (#91)
SoftwareAG/c8y-releasenotes: content/release-10-16-0/platform-10-16-0-bundle/10-16-0-0.md

</tr>

<td>Administration</td>

> <td>Application packages can now be shared with subtenants by setting their availability to SHARED. This way the packages don´t have to be subscribed individually per tenant, and a subtenant can select from a variety of packages if available and shared by their parent or Management tenant.</td>

Reviewer: ```suggestion
<td>Application packages can now be shared with subtenants by setting their availability to SHARED. This way the packages don't have to be subscribed individually per tenant, and a subtenant can select from a variety of packages if available and shared by their parent or Management tenant.</td>

```

MTM-42808</td>
</tr>

<tr>
<td>Notifications 2.0</td>
<td>Introduced the possibility to filter queries on Notifications 2.0 subscriptions by additional fields:
<br>- "subscription" - the subscription name.
<br>- "typeFilter" - the type filter.  This will filter subscriptions by the <code>subscriptionFilter.typeFilter</code> field.

> These new filtering capabilities are available via the REST API and the Java SDK.  See the Cumulocity IoT OpenAPI specification for more details of how to use the new query parameters.</td>

Reviewer 2: ```suggestion
These new filtering capabilities are available via the REST API and the Java SDK. See the Cumulocity IoT OpenAPI specification for more details of how to use the new query parameters.</td>

```

MTM-42808</td>
</tr>

<tr>
<td>Notifications 2.0</td>
<td>Introduced the possibility to filter queries on Notifications 2.0 subscriptions by additional fields:
<br>- "subscription" - the subscription name.
<br>- "typeFilter" - the type filter.  This will filter subscriptions by the <code>subscriptionFilter.typeFilter</code> field.

> These new filtering capabilities are available via the REST API and the Java SDK. See the Cumulocity IoT OpenAPI specification for more details of how to use the new query parameters.</td>

Author: ```suggestion
These new filtering capabilities are available via the REST API and the Java SDK. See the <a href="https://cumulocity.com/api/" class="no-ajaxy">Cumulocity IoT OpenAPI Specification </a> for more details of how to use the new query parameters.</td>

```

MTM-42808</td>
</tr>

<tr>
<td>Notifications 2.0</td>
<td>Introduced the possibility to filter queries on Notifications 2.0 subscriptions by additional fields:
<br>- "subscription" - the subscription name.
<br>- "typeFilter" - the type filter.  This will filter subscriptions by the <code>subscriptionFilter.typeFilter</code> field.

> These new filtering capabilities are available via the REST API and the Java SDK. See the <a href="https://cumulocity.com/api/" class="no-ajaxy">Cumulocity IoT OpenAPI Specification </a> for more details of how to use the new query parameters.</td>

Reviewer: ```suggestion
These new filtering capabilities are available via the REST API and the Java SDK. See the <a href="https://cumulocity.com/api/" class="no-ajaxy">Cumulocity IoT OpenAPI Specification</a> for more details of how to use the new query parameters.</td>

```
